config/share_notify.yml.prod.sample

```diff
@@ -0,0 +1,3 @@
+---
+host: <%= ENV.fetch('APPS_DHH_PROD_SHARE_NOTIFY_HOST') {'https://staging.osf.io'}%>
```

Comment:
@mcritchlow  Yes, that file is needed to copy to the share_notify.yml file depend on the environment like prod or staging.  It uses the correct environment variable name.